sites/all/modules/reol_lazy_image/js/reol_lazy_image.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ * Lazy load images with the 'js-lazy' class.
+ */
+(function($) {
+
+  // Obverser configuration.
+  let options = {
```

Comment:
Will `let` work in all/most browsers?

Reply:
Right that's from EI-11 an up.... so have change them to "var".